t/Chefio.t

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Chefio)],
+    # At a minimum, be sure to include tests for all:
+    # - primary_example_queries
+    # - secondary_example_queries
+    'example query' => test_spice(
```

Comment:
Please update the tests to make sure they're real queries your Instant Answer will handle. Also be sure to add some queries which should not trigger your Instant Answer:

e.g "cheap cookbook" should probably not trigger this, but currently it will.